src/sentry/api/endpoints/group_details.py

```diff
@@ -85,7 +85,7 @@ def _get_activity(self, request: Request, group, num):
 
     def _get_seen_by(self, request: Request, group):
         seen_by = list(GroupSeen.objects.filter(group=group).order_by("-last_seen"))
-        return serialize(seen_by, request.user)
+        return [seen for seen in serialize(seen_by, request.user) if seen is not None]
```

Comment:
Is there a better way to filter out invalid seen_by data than this? I don't want to pollute the API data with `None` values, but didn't see a good way around this in the base serializer

Reply:
seems like you could check the existence of the users in the seen_by list here instead of downstream in the serializer, but not ideal as the serializer could be used in other places. i think this is okay for now